src/Lib/Modes/ItemInsertMode.vala

```diff
@@ -93,6 +111,17 @@ public class Akira.Lib.Modes.ItemInsertMode : AbstractInteractionMode {
             view_canvas.selection_manager.reset_selection ();
             view_canvas.selection_manager.add_to_selection (instance.id);
 
+            // if a path is being inserted, then start the PathEditMode
+            if (item_insert_type == "path") {
+                path_edit_mode = new Akira.Lib.Modes.PathEditMode (view_canvas, true, instance);
```

Comment:
nit: no need for the `Akira.` namespace, you can start directly with `Lib.` since we're already withing the `Akira` namespace.